Doc/library/shutil.rst

```diff
@@ -529,6 +529,9 @@ file then shutil will silently fallback on using less efficient
 
 .. versionchanged:: 3.8
 
+.. versionchanged:: 3.12.1
```

Comment:
I am not sure what the version should be here now.

Reply:
The next one.
```suggestion
.. versionchanged:: 3.14
```